packit/api.py

```diff
@@ -876,3 +877,35 @@ def validate_package_config(working_dir: Path) -> str:
         config_content = load_packit_yaml(config_path)
         v = PackageConfigValidator(config_path, config_content)
         return v.validate()
+
+    def create_sourcegit_from_upstream(
+        self,
+        upstream_url: str,
+        upstream_ref: Optional[str] = None,
+        fedora_package: Optional[str] = None,
+        centos_package: Optional[str] = None,
+        dist_git_branch: Optional[str] = None,
+        dist_git_path: Optional[Path] = None,
```

Comment:
```suggestion
        dist_git_path: Optional[Path] = None,
        dist_git_branch: Optional[str] = None,
        fedora_package: Optional[str] = None,
        centos_package: Optional[str] = None,
```

Reorder this to reflect the actual order of priority.